app/src/main/java/app/revanced/integrations/patches/GeneralAdsPatch.java

```diff
@@ -12,6 +12,7 @@ public final class GeneralAdsPatch extends Filter {
             "home_video_with_context",
             "related_video_with_context",
             "comment_thread", // skip blocking anything in the comments
+            "|comment.", // skip blocking anything in the comments replies
```

Comment:
What do the pathBuilder strings look like for ˋcomment_threadˋ and ˋ|commentˋ?

Reply:
I added ```|comment.``` to define a component name that contains only a reference like that:

```CellType|comment.eml```

In this way, despite the component name doesn't contain ```comment_thread```, it can be ignored as well

Reply:
> What do the pathBuilder strings look like for ˋcomment_threadˋ and ˋ|commentˋ?

For comments reolies, something like that:

```CellType|comment.eml```